autoload/vista/executive/nvim_lsp.vim

```diff
@@ -0,0 +1,70 @@
+" Copyright (c) 2019 Alvaro Muñoz
+" MIT License
+" vim: ts=2 sw=2 sts=2 et
+
+let s:provider = fnamemodify(expand('<sfile>'), ':t:r')
+
+let g:vista_executive_nvim_lsp_reload_only = v:false
+let g:vista_executive_nvim_lsp_should_display = v:false
+let g:vista_executive_nvim_lsp_fetching = v:true
+
+function! s:AutoUpdate(fpath) abort
+  let g:vista_executive_nvim_lsp_reload_only = v:true
+  call s:RunAsync()
+endfunction
+
+function! s:Run() abort
+  call s:RunAsync()
+  while g:vista_executive_nvim_lsp_fetching
+    sleep 100m
+  endwhile
+  return get(s:, 'data', {})
+endfunction
+
+function! vista#executive#nvim_lsp#SetData(data) abort
+  let s:data = a:data
+endfunction
+
+function! s:RunAsync() abort
+    call vista#SetProvider(s:provider)
```

Comment:
Is there any way to check if the built-in LSP support of neovim is avaliable?

Reply:
Sure, I have added a check for nvim-0.5 to make sure both neovim and the minimum right version are used